homeassistant/components/airvisual/__init__.py

```diff
@@ -317,7 +317,7 @@ async def async_migrate_entry(hass, config_entry):
     return True
 
 
-async def async_unload_entry(hass, config_entry):
+async def async_unload_entry(hass, config_entry) -> bool:
```

Comment:
We shouldn't mix random typing into this PR; that should be saved for its own PR.